packages/x-components/src/x-modules/facets-next/entities/__tests__/editable-number-range-filter-entity.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { createLocalVue } from '@vue/test-utils';
+import Vuex, { Store } from 'vuex';
+import { EditableNumberRangeFilter } from '@empathyco/x-types';
+import { createNextEditableNumberRangeFilter } from '../../../../__stubs__/filters-stubs.factory';
+import { RootXStoreState } from '../../../../store/store.types';
+import { facetsNextXStoreModule } from '../../store/module';
+import { EditableNumberRangeFilterEntity } from '../editable-number-range-filter.entity';
+
+describe('testing EditableNumberRangeFilterEntity', () => {
+  function prepareFacetsStore(): Store<RootXStoreState> {
```

Comment:
This method was moved to a utils file in this PR: https://github.com/empathyco/x/pull/73

Maybe it's a good idea

Reply:
Done.